sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -254,7 +254,7 @@ public virtual async Task<RecognizeContentOperation> StartRecognizeContentFromUr
                 SourcePath sourcePath = new SourcePath() { Source = formUri.AbsoluteUri };
                 Response response = await ServiceClient.AnalyzeLayoutAsyncAsync(
                     recognizeContentOptions.Language == null ? (Language?)null : recognizeContentOptions.Language,
-                    recognizeContentOptions.Pages,
+                    recognizeContentOptions.Pages.Count == 0 ? null : recognizeContentOptions.Pages,
```

Comment:
does it break if you just pass `recognizeContentOptions.Pages` to the service call?

Reply:
the URL will look like `https://xx.cognitiveservices.azure.com/formrecognizer/v2.1-preview.2/layout/analyze?Pages=`
the service supports it and the request doesn't fail. 
If I set it to null then the SDK won't set the parameter. I prefer the later in case the service ever decides to start failing on empty parameters

Reply:
If that is the preference for .NET, I am good with that.